API/chat/views.py

```diff
@@ -45,7 +45,7 @@ def get(self, request, type, target, *args, **kwargs):
         channel = get_object_or_404(Channel, name=target)
 
         messages = Message.objects.values(
-            'text', 'username', 'datetime_start', 'typing', 'id', 'datetime_sent'
+            'text', 'username', 'datetime_start', 'typing', 'id', 'datetime_sent', 'message_type'
```

Comment:
Message content is no longer just "text", so `text` field name is inappropriate. Consider changing it.